lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset_facts.py

```diff
@@ -26,9 +26,20 @@
     name:
         description:
             - Limit results to a specific virtual machine scale set
+        required: false
+        default: null
     resource_group:
         description:
             - The resource group to search for the desired virtual machine scale set
+        required: false
+
+    format:
+        description:
+            - What data should be returned?
+        default: 'raw'
+        choices:
+            - 'ansible'
```

Comment:
'ansible' vs 'raw' give some hints and raw is against ansible.. that's definitely not the intention. how about 'traditional' or 'tailored' vs 'raw'.

Reply:
hmm... not sure yet... let's think a bit more and then have some more discussion

Reply:
My thoughts were either `api` or `raw` for the raw API values and maybe `curated` for the Ansible formatted ones. Either way we want the latter to be the default and not raw and that needs to be in place beforehand.